src/common/tracer.h

```diff
@@ -0,0 +1,67 @@
+#ifndef TRACER_H_
+#define TRACER_H_
+
+#define SIGNED_RIGHT_SHIFT_IS 1
+#define ARITHMETIC_RIGHT_SHIFT 1
+
+#include <cstring>
+#include <arpa/inet.h>
+#include <yaml-cpp/yaml.h>
+#include <atomic>
+
+#include <jaegertracing/Tracer.h>
+#include <jaegertracing/net/IPAddress.h>
+#include <jaegertracing/net/Socket.h>
+
+
+namespace jaeger_tracing{
+
+  typedef opentracing::Span Span;
+
+  static std::shared_ptr<opentracing::v3::Tracer> tracer = NULL;
```

Comment:
move to cc file. also NULL -> nullptr

Reply:
I dont need a .cc file for anything should i just make .cc file and one line of including .h file?

Reply:
it is required this is the main containers for all the Spans

Reply:
can you show where it is used in the code?

Reply:
```
opentracing::Tracer::InitGlobal(
        std::static_pointer_cast<opentracing::Tracer>(tracer));
```